### DISCUSSION TITLE
请问苹果刷抖音图片卡怎么办

### DISCUSSION THREAD

**Discussion by Daohuai0913** [kkkgo/PaoPaoDNS#216] [问答和帮助]
具体表现
有时评论区图片刷不出 作者主页的封面刷不出
dns 参考教程外包出去 安卓不卡 苹果换dns或者使用流量不卡

**Comment by kkkgo** [comment]
尝试清空redis_dns_v2.rdb重新部署

**Comment by rgh27** [comment]
我碰到过一种情况，我家有ipv6的环境，ppdns默认ipv6是no，所以访问一些有ipv6的连接就会很慢，要等ipv6超时了才返回ipv4，把ppdns的ipv6设置为yes或者关闭光猫的ipv6就好了。
